### PR TITLE
[FFM-3288] Add Metrics Support

### PR DESCRIPTION
**Pull request by enver-bisevac**
* Introduce metrics api calls
* metricsEnabled added in options

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,29 +19,56 @@ Harness Feature Flags (FF) is a feature management solution that enables users t
...
+echo "Evaluation value for flag $FLAG_KEY with target 'enver': " . $client->evaluate($FLAG_KEY, false);
+$client->close();
 ```
 
+## Running the example
+
+To run the example, open the browser and type:
+
+```
+http://localhost/yourscriptfile.php
```
Comment (bmjen) [inline README.md:70]: Need to replace these instructions on how to set up and run the example... either offline or online example and perhaps with the docker command.

File: src/Harness/CFClient.php [hunk 2]
```diff
@@ -12,11 +12,20 @@
...
 use League\Flysystem\Adapter\Local;
 use League\Flysystem\Filesystem;
 use Cache\Adapter\Filesystem\FilesystemCachePool;
+use OpenAPI\Client\Api\MetricsApi;
 use OpenAPI\Client\Model\Evaluation;
+use OpenAPI\Client\Model\KeyValue;
+use OpenAPI\Client\Model\Metrics;
+use OpenAPI\Client\Model\MetricsData;
+
+const VERSION = "0.0.1";
```
Comment (knagurski) [inline src/Harness/CFClient.php:27]: Also `VERSION` is already defined in the class and doesn't match

File: README.md (hunk 1 above)
Comment (enver-bisevac) [reply README.md:70]: Done

File: src/Harness/CFClient.php [hunk 3]
```diff
@@ -73,32 +85,42 @@ public function __construct(string $sdkKey, Target $target, array $options = [])
...
+        $client = new Client([
+            'handler' => $stack,
+        ]);
+        $this->_apiInstance = new ClientApi($client, $this->_baseConf);
+        $this->_metricsConf = new Configuration();
+        $this->_metricsConf->setHost($this->_eventsUrl);
+        $this->_metricsApi = new MetricsApi($client, $this->_metricsConf);
 
         $item = $this->_cache->getItem("cf_data__{$target->getIdentifier()}");
         $cfData = $item->get();
```
[the remark this answers is not in the document]
Comment (enver-bisevac) [reply src/Harness/CFClient.php:107]: Sure

**Review by bmjen — APPROVED** [review]

**Pull request merged**